INSTALL

```diff
@@ -1190,3 +1191,36 @@ the older HIP C++ library build/linkage. To ensure proper build and linkage
 check that `CMAKE_CXX_COMPILER` and `CMAKE_HIP_COMPILER` are set to the same
 compiler. This is especially important when using an MPI compiler (for example
 crayCC) where some linker flags may get dropped if these two are not identical.
+
+Specific options for Enzyme
+===========================
+MFEM and Enzyme need to be setup for the same LLVM/Clang combination in order to
+work properly. On macOS, for example, one can achieve this with the homebrew
+package manager. First we install Enzyme, which in turn install LLVM as a
+dependency. At the time of writing this is LLVM version 19 and has to be adapted
```

Comment:
Homebrew has now `clang version 20.1.5`, but `enzyme` is still shipped with `ClangEnzyme-19.dylib`.
We could add the way to compile Enzyme locally (`../enzyme`).